packages/portal-proto/src/components/DownloadButtons/DownloadFile.tsx

```diff
@@ -79,6 +82,7 @@ export const DownloadFile: React.FC<DownloadFileProps> = ({
 
   return (
     <DownloadButton
+      data-testid={customDataTestID}
```

Comment:
Is this part working? Are you able to see the data-testid?

Reply:
yes, see above